build.sbt

```diff
@@ -48,13 +48,14 @@ lazy val keeper = project
   .settings(stdSettings("zio-keeper"))
   .settings(
     libraryDependencies ++= Seq(
-      "dev.zio"     %% "zio"             % "1.0.0-RC17",
```

Comment:
We should start introducing a constant for zio versions. :D

Reply:
I added this to https://github.com/zio/zio-keeper/issues/115